sdk/cmake/msvc.cmake

```diff
@@ -559,3 +543,17 @@ function(add_linker_script _target _linker_script_file)
         add_target_property(${_target} LINK_DEPENDS ${_file_full_path})
     endif()
 endfunction()
+
+# handle C++ options
```

Comment:
```suggestion
# Manage our C++ options
```
(the same comment as for gcc.cmake, so that a string search can find both at the same time)